specification/evaluation-context/evaluation-context.json

```
@@ -0,0 +1,23 @@
+[
+    {
+        "id": "Requirement 3.1",
+        "clean id": "requirement_3_1",
+        "content": "The `evaluation context` structure MUST define a required `targeting key` field of type string, identifying the subject of the flag evaluation.",
+        "RFC 2119 keyword": "MUST",
+        "children": []
+    },
+    {
+        "id": "Requirement 3.2",
+        "clean id": "requirement_3_2",
+        "content": "The evaluation context MUST define the following optional fields: `email` (string), `first name` (string), `last name`(string), `name`(string), `ip`(string), `tz`(string), `locale`(string), `country code` (string), `timestamp`(date), `environment`(string), `application`(string), and `version`(string).",
```

```suggestion
        "content": "The evaluation context MUST provide access the following optional attributes: `email` (string), `first name` (string), `last name`(string), `name`(string), `ip`(string), `tz`(string), `locale`(string), `country code` (string), `timestamp`(date), `environment`(string), `application`(string), and `version`(string).",
```

Optional suggestion: Making it clear that you want setters/getters for java, not raw field access.

---

> MUST provide access the following optional attributes

I see what you're getting at, but this suggest read-only to me (access, but not mutation).